### PR TITLE
Add wrapper for export2graphlan

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

I would like to migrate the export2graphlan wrapper that I developed [here](https://github.com/asaim/galaxytools).

Bérénice

### CODE REVIEW

File: tools/export2graphlan/export2graphlan.xml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        <param argument="--abundance_threshold" type="float" value="20.0" label="Minimun abundance value for a clade to be annotated"/>
+        <param argument="--most_abundant" type="integer" label="Number of clades to highlight" optional="True"/>
+        <param argument="--least_biomarkers" type="integer" label="Minimum number of biomarkers to extract" help=" The taxonomy is parsed, and the level is choosen in order to have at least the specified number of biomarkers" optional="True"/>
+        <param argument="--fname_row" type="integer" value="0" label="Row number containing the names of the features" help="-1 specifies that no names are present in the matrix"/>
+        <param argument="--sname_row" type="integer" value="0" label="Row number containing the names of the samples" help="-1 specifies that no names are present in the matrix"/>
+        <param argument="--metadata_rows" type="integer" label="Row number to use as metadata" optional="True"/>
+        <param argument="--skip_rows" type="text" label="Row number to skip from the input file" help="The row numbers ar indexed to 0 and comma separated" optional="True"/>
```
Comment (bgruening) [inline tools/export2graphlan/export2graphlan.xml]: Can this be of type column?

File: tools/export2graphlan/export2graphlan.xml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        <param argument="--def_font_size" type="integer" value="10" label="Default font size"/>
+        <param argument="--min_font_size" type="integer" value="8" label="Minimum font size"/>
+        <param argument="--max_font_size" type="integer" value="12" label="Maximum font size"/>
+        <param argument="--annotation_legend_font_size" type="integer" value="10" label="Font size for the annotation legend"/>
+        <param argument="--abundance_threshold" type="float" value="20.0" label="Minimun abundance value for a clade to be annotated"/>
+        <param argument="--most_abundant" type="integer" label="Number of clades to highlight" optional="True"/>
+        <param argument="--least_biomarkers" type="integer" label="Minimum number of biomarkers to extract" help=" The taxonomy is parsed, and the level is choosen in order to have at least the specified number of biomarkers" optional="True"/>
+        <param argument="--fname_row" type="integer" value="0" label="Row number containing the names of the features" help="-1 specifies that no names are present in the matrix"/>
```
Comment (bgruening) [inline tools/export2graphlan/export2graphlan.xml]: type="data_column" ?

Comment (bebatut) [inline tools/export2graphlan/export2graphlan.xml]: For selecting a row number?

Comment (bgruening) [inline tools/export2graphlan/export2graphlan.xml]: Yes, not a good idea? And to cover the -1 case you can make this param optional? Just an idea.

Comment (bebatut) [inline tools/export2graphlan/export2graphlan.xml]: But is not data_column dedicated to selection of column and not rows?

Comment (bgruening) [inline tools/export2graphlan/export2graphlan.xml]: Blubb, I go to bed ... you are right.

**Pull request merged**